packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -1,21 +1,56 @@
 // @flow
 import * as React from "react";
 
+import {IDProvider} from "@khanacademy/wonder-blocks-core";
+import {type Typography} from "@khanacademy/wonder-blocks-typography";
+
 import FieldHeading from "./field-heading.js";
-import TextField from "./text-field.js";
+import TextField, {type TextFieldType} from "./text-field.js";
 
 type Props = {|
+    /**
+     * An optional unique identifier for the TextField.
+     * If no id is specified, a unique id will be auto-generated.
+     */
+    id?: string,
```

Comment:
This is another change from the implementation spec (will update it). I have now made `id` optional since `IDProvider` is used to generated unique ids.

Reply:
Are there plans to add a `testId` prop eventually as well?

Reply:
Yes. That is in the last [ticket](https://khanacademy.atlassian.net/browse/WB-1069) for the [epic](https://khanacademy.atlassian.net/browse/WB-1055).